### PR TITLE
GR-CRM CEM-2589 Row Actions

### PR DESCRIPTION
**Pull request by B-Fuqua**
Modified the CRM Table component adding an action button to the end of each row
image

### CODE REVIEW

File: src/Containers/CRMTable/CRMTable.tsx [hunk 1]
```diff
@@ -62,6 +66,11 @@ export const CRMTable: React.FC<ICRMTableProps> = ({
             ])
         })
 
+    function justMenuClick(event: React.MouseEvent<SVGElement, MouseEvent>){
```
Comment (ralph-dev) [inline src/Containers/CRMTable/CRMTable.tsx:69]: ES6 notation

**Review by ralph-dev — CHANGES_REQUESTED** [review]

File: src/Containers/CRMTable/CRMTable.tsx [hunk 2]
```diff
@@ -62,6 +66,11 @@ export const CRMTable: React.FC<ICRMTableProps> = ({
             ])
         })
 
+    const justMenuClick = (event: React.MouseEvent<SVGElement, MouseEvent>) => {
```
Comment (ralph-dev) [inline src/Containers/CRMTable/CRMTable.tsx:69]: onMenuClick?
JSDocs missing

**Review by ralph-dev — CHANGES_REQUESTED** [review]

File: src/Containers/CRMTable/CRMTable.tsx (hunk 2 above)
Comment (B-Fuqua) [reply src/Containers/CRMTable/CRMTable.tsx:69]: onMenuClick JSDoc is on line 15, or does justMenuClick need one as well?

**Review by ralph-dev — APPROVED** [review]
Lgtm, makes sense

**Pull request closed without merging**